examples/scripts/ct_astra_datagen_foam2.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# This file is part of the SCICO package. Details of the copyright
+# and user license can be found in the 'LICENSE.txt' file distributed
+# with the package.
+
+r"""
+CT Data Generation for NN Training
+==================================
+
+This example demonstrates how to generate CT synthetic data for
+training neural network models. If desired, a basic reconstruction can be generated
+using filtered back projection (FBP).
```

Comment:
Line breaks look a bit funky.

Reply:
Trivial issue: lines are quite a bit shorter than those in other example scripts